code/modules/power/apc.dm

```diff
@@ -67,6 +67,7 @@
 	var/cell_type = /obj/item/weapon/cell/apc
 	var/opened = 0 //0=closed, 1=opened, 2=cover removed
 	var/shorted = 0
+	var/night_mode = 0 // Determines if the light level is set to dimmed or not

```

Comment:
I forgot to respond but the initial implementation calls for bool states, in future it will be like a dimmer switch.